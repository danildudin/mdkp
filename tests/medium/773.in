28 10 0
659 444 520 79 189 789 633 387 549 166 851 535 514 647 566 490 431 381 987 357 675 205 275 306 530 914 318 454 
592 881 623 951 151 753 527 39 593 927 722 833 795 128 539 365 520 366 961 53 431 334 32 997 434 694 257 481 
670 828 10 423 566 392 923 355 673 237 932 127 803 518 753 170 466 753 561 258 268 405 463 847 940 229 377 505 
721 747 1000 991 77 117 437 507 951 523 153 939 290 361 777 428 169 383 442 106 573 448 277 98 276 749 100 988 
624 587 774 47 736 797 220 175 958 248 814 870 899 942 918 852 919 342 282 736 827 264 680 710 662 88 328 159 
682 542 628 409 713 367 630 276 512 469 34 19 186 621 519 617 724 218 91 18 406 316 52 44 538 621 910 169 
5 855 15 835 692 609 810 335 116 129 884 160 941 421 663 565 771 15 499 941 136 653 440 90 588 219 948 416 
547 562 606 17 391 346 934 573 112 644 226 457 258 975 541 356 146 368 430 138 435 821 797 954 898 790 220 140 
545 939 639 298 482 588 726 278 550 450 129 960 909 301 460 967 990 735 379 642 127 873 528 58 796 906 825 750 
750 278 206 818 63 638 11 808 315 670 408 61 114 271 230 276 457 954 139 641 862 760 339 456 383 729 326 818 
181 166 949 315 315 931 483 145 495 777 492 441 345 902 76 24 435 863 881 468 875 214 383 486 458 877 725 801 
6505 5631 6942 5549 7044 5651 6561 6132 6547 9623 
