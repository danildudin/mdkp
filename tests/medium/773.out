7253
11
5 6 7 10 11 13 17 18 20 25 27 
